src/header/Header.js

```diff
@@ -110,6 +112,7 @@ Header.propTypes = {
   rightContainerStyle: ViewPropTypes.style,
   backgroundColor: PropTypes.string,
   backgroundImage: PropTypes.object,
+  backgroundImageStyle: PropTypes.style,
```

Comment:
There's no PropTypes.style.

Reply:
my bad, I was too quick. do you think simple `PropTypes.object` should be enough?

Reply:
I think ViewPropTypes.style could work or even `Image.propTypes.style`. 

The latter would be more accurate but I am not sure if it works.